src/coreclr/jit/instr.h

```diff
@@ -279,7 +298,6 @@ enum insBarrier : unsigned
     INS_BARRIER_OSHLD =  1,
     INS_BARRIER_OSHST =  2,
     INS_BARRIER_OSH   =  3,
-
```

Comment:
Don't remove this empty line (outside of `TARGET_LOONGARCH64` changes). Why? It's gratuitous.

Reply:
OK, Thanks